miasm2/jitter/arch/JitCore_mips32.c

```diff
@@ -266,6 +269,153 @@ JitCpu_set_cpu(JitCpu *self, PyObject *value, void *closure)
 	return -1;
 }
 
+
+
+static PyObject *
```

Comment:
Instead of copy/paste these functions, can you merge the basic ones in a common file ?

That way, only the arch specifics functions will appear.